packages/grid/x-grid-modules/src/models/gridOptions.tsx

```diff
@@ -176,27 +176,27 @@ export interface GridOptions {
    */
   sortModel?: SortModel;
   /**
-   * Callback fired when the click event comes from a cell element.
+   * Callback fired when a click event comes from a cell element.
    * @param param With all properties from [[CellParams]].
    */
   onCellClick?: (param: CellParams) => void;
   /**
-   * Callback fired when the hover event comes from a cell element.
+   * Callback fired when a hover event comes from a cell element.
    * @param param With all properties from [[CellParams]].
    */
   onCellHover?: (param: CellParams) => void;
   /**
-   * Callback fired when the click event comes from a row container element.
+   * Callback fired when a click event comes from a row container element.
    * @param param With all properties from [[RowParams]].
    */
   onRowClick?: (param: RowParams) => void;
   /**
-   * Callback fired when the hover event comes from a row container element.
+   * Callback fired when a hover event comes from a row container element.
    * @param param With all properties from [[RowParams]].
    */
   onRowHover?: (param: RowParams) => void;
   /**
-   * Callback fired when one row get selected.
+   * Callback fired when one row is selected.
```

Comment:
One, and only one row?

Reply:
I propose we remove this prop in https://github.com/mui-org/material-ui-x/issues/247